pulpcore/tests/functional/api/test_crd_artifacts.py

```diff
@@ -135,11 +136,10 @@ def test_all(self):
         files = {'file': utils.http_get(FILE_URL)}
         artifact = api_client.post(ARTIFACTS_PATH, files=files)
         self.addCleanup(api_client.delete, artifact['_href'])
-        sudo = () if cli.is_root(cfg) else ('sudo',)
-        cmd = sudo + ('ls', artifact['file'])
-        cli_client.run(cmd)
+        cmd = ('ls', artifact['file'])
+        cli_client.run(cmd, sudo=True)
```

Comment:
Does this change makes sudo mandatory where before it was optional?

Reply:
No. This logic to verify if an user was root and add sudo if necessary was moved to the CLI client itself.

See: https://github.com/PulpQE/pulp-smash/commit/d97a6b383321a8c082593e3b9af503657a9862e1